.travis.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       env: PROTOBUF_C_EXT=true
     - php: '7.2'
       install:
-      - travis_retry pecl install protobuf-3.6.1
+      - travis_retry pecl install protobuf-3.12.2
```

Comment:
This line can be left as is, the intent is to make sure `composer install` fails with an incompatible version of the protobuf extension, otherwise LGTM!

Reply:
Done.